### ISSUE TITLE
Feature Request: dynamic entity retrieval in the spirit of state.get()

### ISSUE THREAD

**Issue by wsw70** [custom-components/pyscript#109]
In the same spirit as `state.get()`, it would be good to have `entity.get()` (and the rest of the family) that would allow a dynamic access to the attributes of an entity, when the entity is available as a string, or otherwise computed.

---

This is a followup on https://github.com/custom-components/pyscript/issues/102, where @dlashua provided a solution to access group attributes via 

```
for group_entity_id in group.michael.entity_id:
  log.info(group_entity_id)
```

The proposal would solve the problem where `group.michael` is available as a the string `"group.michael"`. My current workaround is to use `eval("group.michael")` which does not seem to be a reasonable solution.

**Comment by craigbarratt** [comment]
I think you are asking about entity (state) attributes.  A specific attribute is available via `state.get()`, and all attributes are available vis `state.getattr()`.

Example: `state.get("group.michael.entity_id")` and `state.getattr("group.michael")`.

You can also set a specific attribute via direct assignment, or using `state.setattr()`.  If you want to set all the attributes use `state.set()`.

**Comment by wsw70** [comment]
@craigbarratt  Thank you, I learned something today. I now actually realized that HA has 'domains' that can be of many kind. 

I thought that this was just something "tangible" such as `switch` (and therefore I know how to get their states) but I did not completely absorbed the fact that can also be `group` - and getting their attributes (and specifically `entity_id`) will give me the content of the group.

retrospectively  - this makes sense as `group.something` is part of the dev tool state table.

@dlashua thank you very much to you too for the explanation in https://github.com/custom-components/pyscript/issues/102 - it is now clear.

And of course my `entity.get()` does not make sense.

Ah la la, this is a lot of "Issues" uselessly opened - sorry for that.
